modelling_earth/plot.py

```diff
@@ -104,12 +104,15 @@ def _plot_swarm_2d(swarm, ax, **kwargs):
 
 def save_plots_2d(
     dataset,
+    swarm,
```

Comment:
This looks really nice @aguspesce! I would make a little change. I would set the `swarm` argument as optional (with default to `None`) and remove the `plot_swarm` argument. So, in case `swarm` is `None` you won't plot it, otherwise you will.